ios/main.mm

```diff
@@ -92,16 +92,20 @@ kern_return_t catch_exception_raise(mach_port_t exception_port,
 static float g_safeInsetTop = 0.0;
 static float g_safeInsetBottom = 0.0;
 
+// We no longer need to judge if jit is usable or not by according to the ios version.
```

Comment:
Why, because there no longer are any relevant versions that allow JIT?

Reply:
We do not acturally have such code now

Reply:
Now we only use get_debugged()

Reply:
Ah right. I haven't really been keeping up with the iOS changes too much.